go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/k0sproject/k0s
 
-go 1.23
+go 1.23.0
```

Comment:
Shouldn't it be 1.23.5 instead of 1.23.0?

Reply:
No, this is more about the minimum required Go version. I'd rather avoid to update this for every Go patch version bump.

Reply:
Ah fair enough